DAQ_System/Source/strainGauges.cpp

```diff
@@ -0,0 +1,71 @@
+#include "mbed.h"
+#include "Hx711.h"
+
+#include <cstdint>
+#include <ios>
+#include <iostream>
+
+#define kFactor 2 // it's typcally 2 
+#define dout ARDUINO_UNO_D2
+#define sclk ARDUINO_UNO_D3
+#define Vs   3.3 // input voltage is anywhere between 2.7 - 5.5
```

Comment:
Remove these unused variables.
Move `Vs` and `stepSize` into the inside of the scope of the main function.
Rename `Vs` to something more explicit, such as `kVoltageSupply`, and use `static constexpr` instead of `#define`